src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -123,7 +128,16 @@ public Person toModelType() throws IllegalValueException {
         }
         final Birthday modelBirthday = new Birthday(birthday);
 
-        return new Person(modelName, modelPhone, modelEmail, modelAddress, modelRemark, modelTags, modelBirthday);
+        if (moneyOwed == null) {
```

Comment:
This part is saying if `moneyOwed` in the json is null, throw an error. But every time we add in new attributes, it'll be null cuz it's missing, so I think it'll make the all contacts in the address book disappear.

![image](https://github.com/AY2324S2-CS2103T-T16-2/tp/assets/40530851/c7d9dc7e-e41c-470e-99e4-af24735d2f81)

Reply:
Made use of Optional.ofNullable here for the changes